.devcontainer/devcontainer.json

```
@@ -0,0 +1,21 @@
+{
+	"name": "Godot Dev-Container",
+	"build":{
+		"dockerfile": "Dockerfile"
+	},
+	
+	"customizations": {
+		"vscode": {
+			"settings": {
+				"extensions.ignoreRecommendations": true
+			},
+			"extensions": [
+				"ms-azuretools.vscode-docker",
+				"ms-vscode.cpptools"
+			]
+		}
+	}
+
+	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
+	// "remoteUser": "root"
```

```suggestion
	//"remoteUser": "root"
```
Commented out code has no space in front of it to differentiate it

---

I removed the comment altogether. Root should not be needed.